lib/ex_sdp/attribute/fmtp.ex

```diff
@@ -98,11 +104,14 @@ defmodule ExSDP.Attribute.FMTP do
 
   @spec parse(binary()) :: {:ok, t()} | {:error, reason()}
   def parse(fmtp) do
-    with [pt_string, rest] <- String.split(fmtp, " "),
+    with [pt_string | rest] <- String.split(fmtp, " "),
          {:ok, pt} <- Utils.parse_payload_type(pt_string) do
-      rest
-      |> String.split(";")
-      |> do_parse(%__MODULE__{pt: pt})
+      params =
+        for param <- rest do
+          String.replace(param, ";", "")
+        end
+
+      do_parse(params, %__MODULE__{pt: pt})
```

Comment:
FMTP parser has already been prepared for receiving multiple attributes but we expect them to be separated with `;`. Take a look at our tests: https://github.com/membraneframework/ex_sdp/blob/master/test/ex_sdp/attribute/fmtp_test.exs#L8

When you run `mix test` with your changes, some of them won't pass

However, looking at the RFC 7587, indeed it's possible to have multiple attributes separated both with `;` and `space` :thinking: 

I would propose the following code 

```suggestion
    with [pt_string, rest] <- String.split(fmtp, " ", parts: 2),
             {:ok, pt} <- Utils.parse_payload_type(pt_string) do
      rest
      |> String.split(";")
      # remove leading whitespaces
      |> Enum.map(&String.trim(&1))
      |> do_parse(%__MODULE__{pt: pt})
```     

We also need to add a test for a line with spaces and `maxptime` and `ptime` attributes

```
a=fmtp:121 maxplaybackrate=16000; sprop-maxcapturerate=16000; maxptime=20; ptime=20; maxaveragebitrate=24000; cbr=0; useinbandfec=0; usedtx=0


```